lib/pengine/complex.c

```diff
@@ -180,14 +180,6 @@ get_meta_attributes(GHashTable * meta_hash, pcmk_resource_t * rsc,
         .op_data = NULL
     };
 
-    if (node) {
```

Comment:
Add a COMPAT comment stating that `node` is unused, as a reminder to drop it if/when we pull this internal

Reply:
I'll add a doxygen block noting that node is ignored. This function will eventually be deprecated entirely and replaced with a new one.